Commands/Rubocop.tmCommand

```diff
@@ -12,6 +12,8 @@ if [ -z "$TM_RUBY" ]; then
     exit 1
 fi  
 
+cd "$TM_PROJECT_DIRECTORY"
+export TM_RUBOCOP_RUBY_VERSION=2.2
```

Comment:
Disclaimer: this is the first time I touch any textmate bundles.

The only other options I can think of for this  config is either set the environment variable in the system, or in textmate preference->variables.  The system-wide one is out of question since that requires user to do something outside textmate.  Between using textmate variable and hardcoded in script, I went with the later because the script is easily accessible with bundle editor.  

I pushed a change just now to allow TM_RUBOCOP_RUBY_VERSION to be set by user (either in system shell or in textmate environment).  Please let me know if you have better means of setting this option.